awx/main/management/commands/inventory_import.py

```diff
@@ -853,8 +918,19 @@ def _create_update_group_hosts(self):
                         logger.debug('Host "%s" already in group "%s"', db_host.name, db_group.name)
                     for db_host in db_hosts_qs.exclude(groups__id=db_group.id):
                         self._batch_add_m2m(db_group.hosts, db_host)
+                        group_host_additions += 1
                         logger.debug('Host "%s" added to group "%s"', db_host.name, db_group.name)
                 self._batch_add_m2m(db_group.hosts, flush=True)
+
+        if group_host_additions:
+            logger.info('Added {} group host memberships out of {} from source'.format(
+                group_host_additions, group_host_count
```

Comment:
I'm sure it would be better to combine these messages with the membership removal messages...

but then you get to the "why not just make a table?" argument, and these are log messages, so... not really appropriate for the medium.